event_operating_unit/views/event_registration_views.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_event_registration_tree_inherit_ou">
+        <field name="name">event.registration.tree</field>
+        <field name="model">event.registration</field>
+        <field name="inherit_id" ref="event.view_event_registration_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="field[@name='state']" position="after">
+                <field name="operating_unit_id" groups="operating_unit.group_multi_operating_unit" />
+            </xpath>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_registration_search_inherit_ou">
+        <field name="name">event.registration.search</field>
+        <field name="model">event.registration</field>
+        <field name="inherit_id" ref="event.view_registration_search" />
+        <field name="arch" type="xml">
+            <search>
+                <field name="operating_unit_id" string="Operating Unit" groups="operating_unit.group_multi_operating_unit" />
+            </search>
+            <xpath expr="//group[1]">
+                <filter string="Operating Unit" context="{'group_by': 'operating_unit_id'}" groups="operating_unit.group_multi_operating_unit" />
```

Comment:
does this work?

Reply:
I tesed it on the event. it works.

Reply:
is this event?

Reply:
If the field isn't stored I doubt it works :/